src/webgpu/api/operation/command_buffer/copies.spec.ts

```diff
@@ -209,3 +414,194 @@ g.test('b2t2t2b').fn(async t => {
 
   t.expectContents(dst, data);
 });
+
+// TODO(jiawei.shao@intel.com): support all WebGPU texture formats
+g.test('t2t_non_compressed_color_formats')
+  .params(
+    params()
+      .combine(
+        poptions('textureSize', [
+          { width: 32, height: 32, depth: 3 },
```

Comment:
It would be nice to have the texture sizes of src and dst not have to be the same (including more elongated textures and depth = 1 and a varying number). Also it would be good to test copying the lowest mip level.